lib/transactions/state-store.mjs

```
@@ -0,0 +1,91 @@
+      await delay(retryMs);
+    }
+  }
+
+  try {
+    return await operation();
+  } finally {
+    fs.closeSync(handle);
+    try {
+      fs.unlinkSync(lockPath);
+    } catch (error) {
+      if (error.code !== "ENOENT") throw error;
```

Biome (2.5.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)